x/supplier/keeper/msg_server_create_claim.go

```diff
@@ -13,7 +16,9 @@ func (k msgServer) CreateClaim(goCtx context.Context, msg *suppliertypes.MsgCrea
 	// TODO_BLOCKER: Validate the signature on the Claim message corresponds to the supplier before Upserting.
 
 	ctx := sdk.UnwrapSDKContext(goCtx)
-	logger := k.Logger(ctx).With("method", "CreateClaim")
+	// TODO_DISCUSS_IN_THIS_COMMIT: log lines from the cosmos-sdk logger aren't printed in tilt for some reason.
```

Comment:
After more careful examination, what I was observing was the fact that our localnet [config.toml](https://github.com/pokt-network/poktroll/blob/main/localnet/poktrolld/config/config.toml#L53) is currently set to the `"info"`, by default. I also find it worthwhile to point out that is is also set to `"plain"` format, whereas we could set it to JSON, which may simplify downstream log aggregation, parsing, searching, etc. (in addition to being more consistent with the default output format of our off-chain logs):

![image](https://github.com/pokt-network/poktroll/assets/600733/1d6ad976-5f81-4264-9925-04e8b2d60a19)

![image](https://github.com/pokt-network/poktroll/assets/600733/13eb86dc-f5e0-4bc9-b53b-10d6f29dea32)

Yields:

![image](https://github.com/pokt-network/poktroll/assets/600733/7a9fe299-4167-4a1d-8382-19e00c1382d5)

When setting the log level to `"debug"` the log output becomes much more verbose but with cosmos-sdk internal output (which is not typically useful, especially for debugging business logic). For this reason, I **would not** suggest that we change the localnet config to the `"debug"` level. However, this puts us in a position where we have to either A) manually change the config and wade through the excess output in order to effectively use debug logs that we can leave in the code, or to B) use an alternative logger which provides the debugging degree of freedom without including all the cosmos-sdk specific debug log lines. The latter option puts us in a position of inconsistency with respect to which logging library to use so long as we want to keep the debug lines in the code, which we should definitely have the option to do.

---

I'm also noticing that we don't seem to be able to take advantage of tilt's log level filtering (it hasn't a clue):

![image](https://github.com/pokt-network/poktroll/assets/600733/504717a0-112d-41dc-8b28-e9ec8c6de658)

I'm not sure if this has to do with whether logs are printed to stderr vs. stdout or if it's something we can configure tilt to be sensitive to but it would be extremely convenient to be able to use this feature.